packages/twenty-docs/docs/start/self-hosting/docker-compose.mdx

```diff
@@ -6,12 +6,14 @@ sidebar_custom_props:
 ---
 # Step by step instructions:
 
-1) Copy the [.env.example](https://github.com/twentyhq/twenty/blob/main/packages/twenty-server/.env.example) into a `.env` in the same directory where your `docker-compose.yml` file will be
-2) Run the command `openssl rand -base64 32` three times, make note of the string for each
-3) Replace the three "replace_me_with_a_random_string_access" with the three random strings you just generated.
-4) Create a `docker-compose.yml` file from the example below.
-5) Run the command `docker-compose up -d`
-6) Go to http://localhost:3001 and see your docker instance.
+1. Copy the [.env.example](https://github.com/twentyhq/twenty/blob/main/packages/twenty-server/.env.example) into a `.env` in the same directory where your `docker-compose.yml` file will be
+2. Run the command `openssl rand -base64 32` three times, make note of the string for each
+3. Replace the three "replace_me_with_a_random_string_access" with the three random strings you just generated.
+    1. If Twenty will be running behind reverse proxy, add `SERVER_URL=https://<your-api-url.com>` to your `.env`
+4. Create a `docker-compose.yml` file from the example below.
+    1. If Twenty will be running behind reverse proxy, uncomment `SERVER_URL=${SERVER_URL}` in your `docker-compose.yml`
+5. Run the command `docker-compose up -d`
+6. Go to http://localhost:3001 and see your docker instance.
```

Comment:
Maybe add a new h2 : Running behind a reverse proxy ? It feels those sub headings do not belong to the "regular" steps but because it's so common should follow the instructions as a dedicated h2 section.

Reply:
We could also add a "Troubleshooting" section and repeat this process (or redirect to the dedicated h2 section) 

It never hurts to repeat ourselves in documentation.